src/Microsoft.DotNet.Wpf/src/Shared/MS/Internal/ReflectionUtils.cs

```diff
@@ -0,0 +1,131 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable enable
+
+using System.Runtime.CompilerServices;
+using System.Reflection;
+using System.Text;
+using System;
+
+namespace MS.Internal
+{
+    /// <summary>
+    /// Provides utilities for working with reflection efficiently.
+    /// </summary>
+    internal static class ReflectionUtils
+    {
+#if !NETFX
```

Comment:
Is this needed for PBT or something that compiles for .NET Framework? I would love to keep the codebase compilable for .NET FW but I think that ship has already sailed, so if there is no need then you might as well just keep the .NET Core path only.

Reply:
Yeah, I wish I didn't need to but it was the most concise thing to do. Trust me, I'd remove any `netfx` reference if I could, haha.

#9739 needs a dummy fallback for PBT